shaders/EngulfEffect.gdshader

```diff
@@ -0,0 +1,32 @@
+shader_type spatial;
+
+// Set to 0 to disable wiggle
```

Comment:
Are these duplicated from  the membrane shader? If so I'd appreciate a comment here saying so and another in the membrane shader so that if someone edits the variables / the shader they need to edit both shaders.